instrumentation/opentelemetry-api-metrics-1.0/javaagent/src/test/groovy/MeterTest.groovy

```diff
@@ -1,271 +0,0 @@
-/*
- * Copyright The OpenTelemetry Authors
- * SPDX-License-Identifier: Apache-2.0
- */
-
-import static io.opentelemetry.sdk.metrics.data.MetricDataType.DOUBLE_GAUGE
-import static io.opentelemetry.sdk.metrics.data.MetricDataType.DOUBLE_SUM
-import static io.opentelemetry.sdk.metrics.data.MetricDataType.SUMMARY
-import static java.util.concurrent.TimeUnit.SECONDS
-
-import com.google.common.base.Stopwatch
-import io.opentelemetry.api.common.AttributeKey
-import io.opentelemetry.api.common.Attributes
-import io.opentelemetry.api.metrics.AsynchronousInstrument
-import io.opentelemetry.api.metrics.GlobalMeterProvider
-import io.opentelemetry.api.metrics.common.Labels
-import io.opentelemetry.instrumentation.test.AgentInstrumentationSpecification
-import io.opentelemetry.sdk.metrics.data.MetricData
-import io.opentelemetry.sdk.metrics.data.PointData
-import java.util.function.Consumer
-
-class MeterTest extends AgentInstrumentationSpecification {
-
-  def "test counter #builderMethod bound=#bind"() {
-    given:
-    // meters are global, and no way to unregister them, so tests use random name to avoid each other
-    def instrumentationName = "test" + new Random().nextLong()
-
-    when:
-    def meter = GlobalMeterProvider.getMeter(instrumentationName, "1.2.3")
-    def instrument = meter."$builderMethod"("test")
-      .setDescription("d")
-      .setUnit("u")
-      .build()
-    if (bind) {
-      instrument = instrument.bind(Labels.empty())
-    }
-    if (bind) {
-      instrument.add(value1)
-      instrument.add(value2)
-    } else {
-      instrument.add(value1, Labels.of("q", "r"))
-      instrument.add(value2, Labels.of("q", "r"))
-    }
-
-    then:
-    def metricData = findMetric(instrumentationName, "test")
-    metricData != null
-    metricData.description == "d"
-    metricData.unit == "u"
-    metricData.type == expectedType
-    metricData.instrumentationLibraryInfo.name == instrumentationName
-    metricData.instrumentationLibraryInfo.version == "1.2.3"
-    points(metricData).size() == 1
-    def point = points(metricData).iterator().next()
-    if (bind) {
-      point.attributes == Attributes.of(AttributeKey.stringKey("w"), "x", AttributeKey.stringKey("y"), "z")
```

Comment:
Bind binds with empty labels above. I have no idea what's going on with this line and am glad the new tests are easier to understand